test/start-kubernetes.sh

```diff
@@ -29,7 +29,7 @@ KVM_CPU_OPTS="${KVM_CPU_OPTS:-\
  -machine pc,accel=kvm,nvdimm=on}"
 EXTRA_QEMU_OPTS="${EXTRA_QWEMU_OPTS:-\
  -object memory-backend-file,id=mem1,share=${TEST_PMEM_SHARE},\
-mem-path=/data/nvdimm0,size=$((${TEST_PMEM_MEM_SIZE} * 1024 * 1024 + ${TEST_PMEM_LABEL_SIZE})) \
+mem-path=/data/nvdimm0,size=$((${TEST_PMEM_MEM_SIZE} * 1024 * 1024 + ${TEST_PMEM_LABEL_SIZE})),align=2M \
```

Comment:
Have you tried this locally?

If you rely on the CI to try out changes like this, then progress will be very slow.

Reply:
yes you are right, this was not the cause. I then switched to local dev for this, to address overall sizes and alignment issues.

Reply:
I removed that change as it seemed not relevant after all, probably we get 2M align by default?